packages/gasket-cli/lib/commands/create.js

```diff
@@ -0,0 +1,194 @@
+/* eslint-disable max-statements */
+const ora = require('ora');
+const chalk = require('chalk');
+const makeCreateContext = require('../scaffold/create-context');
+const dumpErrorContext = require('../scaffold/dump-error-context');
+const {
+  mkDir,
+  loadPreset,
+  cliVersion,
+  globalPrompts,
+  setupPkg,
+  writePkg,
+  installModules,
+  linkModules,
+  loadPkgForDebug,
+  promptHooks,
+  createHooks,
+  postCreateHooks,
+  generateFiles,
+  writeGasketConfig,
+  applyPresetConfig,
+  printReport
+} = require('../scaffold/actions');
+
+/**
+ * Parses comma separated option input to array
+ *
+ * @param {String} input - option argument
+ * @returns {String[]} results
+ */
+const commasToArray = input => input.split(',').map(name => name.trim());
+
+const createCommand = {
+  id: 'create',
+  description: 'Create a new Gasket application',
+  args: [
+    {
+      name: 'appname',
+      description: 'Name of the Gasket application to create',
+      required: true
+    }
+  ],
+  options: [
+    {
+      name: 'presets',
+      short: 'p',
+      description: `Initial Gasket preset(s) to use.
+      Can be set as short name with version (e.g. --presets nextjs@^1.0.0)
+      Or other (multiple) custom presets (e.g. --presets my-gasket-preset@1.0.0.beta-1,nextjs@^1.0.0)`,
+      parse: commasToArray
+    },
+    {
+      name: 'plugins',
+      description: `Additional plugin(s) to install. Can be set as
+      multiple flags (e.g. --plugins jest --plugins zkconfig@^1.0.0)
+      comma-separated values: --plugins=jest,zkconfig^1.0.0`,
+      parse: commasToArray
+    },
+    {
+      name: 'package-manager',
+      description: `Selects which package manager you would like to use during
+      installation. (e.g. --package-manager yarn)`
+    },
+    {
+      name: 'require',
+      short: 'r',
+      description: 'Require module(s) before Gasket is initialized',
+      parse: commasToArray
+    },
+    {
+      name: 'bootstrap',
+      description: '(INTERNAL) If provided, skip the bootstrap phase',
+      default: false,
+      type: 'boolean',
+      hidden: true
+    },
+    {
+      name: 'generate',
+      description: '(INTERNAL) If provided, skip the generate phase',
+      default: false,
+      type: 'boolean',
+      hidden: true
+    },
```

Comment:
This behavior is inverted now.
```
> gasket create dummy --bootstrap
⚠ Bootstrap phase skipped.
✖ Load package.json for debug. 🐲 Here be dragons!
```

With the oclif version ([code](https://github.com/godaddy/gasket/blob/1b34afeba97be5d2793c9591f69c20300cd4ef22/packages/gasket-cli/src/commands/create.js#L137-L148)), default were `true` and using `--no-bootstrap` would disable.

```
> gasket create dummy --no-bootstrap
error: unknown option '--no-bootstrap'
```

You can use `--no-*` with commander ([docs](https://www.npmjs.com/package/commander#other-option-types-negatable-boolean-and-booleanvalue)). I think it makes sense to only have the nos for these honestly.
```suggestion
    {
      name: 'no-bootstrap',
      description: '(INTERNAL) If provided, skip the bootstrap phase',
      type: 'boolean',
      hidden: true
    },
    {
      name: 'no-generate',
      description: '(INTERNAL) If provided, skip the generate phase',
      type: 'boolean',
      hidden: true
    },
```

Which means the options logic below will need adjusting also
```
const { bootstrap, generate } = options;

    if (bootstrap) {
      await bootstrap(context);
    } else {
      ora('Bootstrap phase skipped.').warn();
      if (generate) {
        await loadPkgForDebug(context);
      }
    }

    if (generate) {
```